android/src/test/java/com/sailthru/mobile/rnsdk/RNSailthruMobilePackageTest.java

```diff
@@ -5,8 +5,6 @@
 import com.facebook.react.bridge.JavaScriptModule;
 import com.facebook.react.bridge.NativeModule;
 import com.facebook.react.bridge.ReactApplicationContext;
-import com.sailthru.mobile.rnsdk.RNSailthruMobileModule;
-import com.sailthru.mobile.rnsdk.RNSailthruMobilePackage;
```

Comment:
Hey Ian! Just wondering how come these are removed? Do we not need them anymore since we have the `@SuppressWarnings` 🧐

Reply:
I think they must have been added automaticlly when I changed the package from com.reactlibrary to com.sailthru.mobile.rnsdk. The tests classes were temporarily in a different package so needed to import the classes manually. Since the tests are normally in the same package in the test directory these imports aren't required.